test/unit/test_mfa_no_cache.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2012-2020 Snowflake Computing Inc. All right reserved.
+#
+
+import json
+
+import pytest
+from mock import patch
+
+import snowflake.connector
+
+try:
+    from snowflake.connector.compat import IS_LINUX  # NOQA
+    from snowflake.connector.options import installed_keyring
+except ImportError:
+    IS_LINUX = True
```

Comment:
Okay, I just remembered what's happening here. The fallback is here for the same reason as `IS_MACOS` (for old driver test). However, since `IS_LINUX` and `IS_MACOS` are not added into repo at the same time, i.e. `IS_MACOS` is relatively new, `IS_LINUX` may not break old driver test as `IS_MACOS`. I was doing the test in one of my test file and it turned out the `IS_LINUX` is old enough. Let me remove the fallback for `IS_LINUX` while keep `IS_MACOS` fallback.